garcon/activity.py

```diff
@@ -40,6 +41,7 @@ def run(self):
 
         if packed_context:
             context = json.loads(packed_context)
+            self.set_log(context)
```

Comment:
Looking more, this might just be `set_log_context`, because the log is already available, we're just adding more data into it. No need for a change if you think `set_log` is fine.